timestampvm/src/api/static_handlers.rs

```diff
@@ -2,6 +2,7 @@
 //! To be served via `[HOST]/ext/vm/[VM ID]/static`.
 
 use crate::vm;
+use avalanche_types::subnet::rpc::snow::engine::common::appsender::AppSender;
```

Comment:
I know this is a comment for `avalanche-types`, but it would probably be a little nicer from a DX perspective to re-export types that are commonly used to the crate-root. 

I'm guessing that a good rule-of-thumb would be any types that are used in this tutorial specifically are pretty common?